gradient/api_sdk/clients/deployment_client.py

```diff
@@ -59,10 +57,13 @@ def start(self, deployment_id):
             gradient deployments start --id <your-deployment-id>
 
         :param str deployment_id: Deployment ID
+        :param bool use_vpc:
         """
-        return self._get_post_response(deployment_id)
 
-    def stop(self, deployment_id):
+        repository = repositories.StartDeployment(api_key=self.api_key, logger=self.logger)
+        repository.start(deployment_id, use_vpc=use_vpc)
```

Comment:
no return here?

Reply:
nope. An exception is raised when command failed